ras_party/views/respondent_view.py

```diff
@@ -99,6 +93,22 @@ def get_respondent_by_email():
     return jsonify(response)
 
 
+@respondent_view.route('/respondents/email', methods=['DELETE'])
+def delete_respondent_by_id():
```

Comment:
minor :- given the mismatch between use of id in the name and email it may be worth a comment ?

Reply:
Done